handbook/engineering/career-development/talent-review-process.md

```diff
@@ -22,7 +22,7 @@ Less frequent than quarterly raises the stakes too high: “I just started this
 
 Prior to the calibration meeting, each <a href="../roles#engineering-manager"><abbr title="Engineering Manager">EM</abbr></a> should evaluate each <a href="../roles#software-engineer"><abbr title="Individual Contributor">IC</abbr></a> reporting to them using the [level descriptions](framework.md#levels) and determine if they are meeting/exceeding/struggling-to-meet the expectations of their given level. EMs should prepare a _short_ statement to justify this, which they will share during the calibration meeting. If they believe the IC to be performing at a higher level, they should also ensure that the IC is meeting the expectations for that higher level, as well as write up a brief promotion pitch. (Note that compensation increases within a level, such as from the start of the compensation band to the midpoint, do not need to go through this process; those can happen any time and are between the EM and their <a href="../roles#director-of-engineering">director</a> to decide.)
 
-This pre-work is greatly aided by [360º feedback](../../people-ops/review-cycles/index.md), so we schedule the talent review accordingly. (Since we do 360º feedback every 6 months, it would only be available for every other talent review.)
+This pre-work is greatly aided by [Impact review cycles](../../people-ops/impact-reviews.md), so we schedule the talent review accordingly. (Since we do 360º feedback every 6 months, it would only be available for every other talent review.)
```

Comment:
@jeanduplessis @felixfbecker I've updated our Impact review cycles link in place of the previous 360 links. Let me know how the talent review will be affected by simultaneous reviews going off during different parts of the year, and if you'd like my help in documenting that here.

Reply:
Yeah the wording here now no longer makes sense, we need to update this. We wouldn't change the scheduling anymore, but we can still note that the most recent 360º feedback is useful.